test/e2e/common/pod_resize.go

```diff
@@ -49,7 +49,11 @@ const (
 	CgroupMemLimit  string = "/sys/fs/cgroup/memory/memory.limit_in_bytes"
 
 	PollInterval time.Duration = 2 * time.Second
-	PollTimeout  time.Duration = time.Minute
+	// PollTimeout  time.Duration = time.Minute
+	PollTimeout  time.Duration = 5 * time.Minute
```

Comment:
Why do we need such a big timeout? And we should not leave commented out code like the line above.

Reply:
Ok, but tests on my machine will have not enough time to sync when `PollTimeout = time.Minute`.

Reply:
Will clean the code.